### ISSUE TITLE
Some systems failed to build for Quicklisp dist

### ISSUE THREAD

**Issue by quicklisp** [tfeb/tfeb-lisp-hax#4]
Building with SBCL 2.2.7.28-02bc916fd / ASDF 3.3.5 for quicklisp dist creation.

Trying to build commit id 65e7379ab0d340ec77381f33d168a29a0a280378

*org.tfeb.hax.metatronic* fails to build with the following error:

```
Unhandled UNBOUND-VARIABLE in thread #<SB-THREAD:THREAD "main thread" RUNNING {10016D0003}>: The variable ORG.TFEB.TOOLS.REQUIRE-MODULE::*MODULE-PATH-DESCRIPTIONS* is unbound.

```

*org.tfeb.hax.simple-loops* fails to build with the following error:

```
Unhandled UNBOUND-VARIABLE in thread #<SB-THREAD:THREAD "main thread" RUNNING {10016D0003}>: The variable ORG.TFEB.TOOLS.REQUIRE-MODULE::*MODULE-PATH-DESCRIPTIONS* is unbound.

```

*org.tfeb.hax.slog* fails to build with the following error:

```
Unhandled UNBOUND-VARIABLE in thread #<SB-THREAD:THREAD "main thread" RUNNING {10016D0003}>: The variable ORG.TFEB.TOOLS.REQUIRE-MODULE::*MODULE-PATH-DESCRIPTIONS* is unbound.

```

*org.tfeb.hax.spam* fails to build with the following error:

```
Unhandled UNBOUND-VARIABLE in thread #<SB-THREAD:THREAD "main thread" RUNNING {10016D0003}>: The variable ORG.TFEB.TOOLS.REQUIRE-MODULE::*MODULE-PATH-DESCRIPTIONS* is unbound.

```

*org.tfeb.hax.utilities* fails to build with the following error:

```
Unhandled UNBOUND-VARIABLE in thread #<SB-THREAD:THREAD "main thread" RUNNING {10016D0003}>: The variable ORG.TFEB.TOOLS.REQUIRE-MODULE::*MODULE-PATH-DESCRIPTIONS* is unbound.

```

*org.tfeb.hax* fails to build with the following error:

```
Unhandled UNBOUND-VARIABLE in thread #<SB-THREAD:THREAD "main thread" RUNNING {10016D0003}>: The variable ORG.TFEB.TOOLS.REQUIRE-MODULE::*MODULE-PATH-DESCRIPTIONS* is unbound.

```

*org.tfeb.hax/test* fails to build with the following error:

```
Unhandled UNBOUND-VARIABLE in thread #<SB-THREAD:THREAD "main thread" RUNNING {10016D0003}>: The variable ORG.TFEB.TOOLS.REQUIRE-MODULE::*MODULE-PATH-DESCRIPTIONS* is unbound.

```

[Full log here](http://report.quicklisp.org/2022-11-01/failure-report/tfeb-lisp-hax.html)

**Comment by tfeb** [comment]
This is happening because you've previously compiled some or all of the code with `org.tfeb.tools.require-module` loaded.  That's caused stuff to end up in the FASLs which makes no sense when it's not loaded, due to reader conditionals.

Almost certainly the underlying cause is that you compiled `org.tfeb.dsm` previously, which *does* (did) depend on `org.tfeb.tools.require-module` as well as some of `org.tfeb.hax`.

Probably it would be theoretically better to try and build each system in a really clean environment with no FASLs lying around but I can see that that's probably absurdly hard in practice

So I've changed `org.tfeb.dsm` so it no longer depends on `org.tfeb.tools.require-module`.  That means that this problem should now go away I think.  So this should be fixed even though there are no changes to this repo that fix it.

So even though I've made no changes
